lib/daru/view/adapters/googlecharts/data_table_iruby.rb

```diff
@@ -38,6 +39,15 @@ def initialize(options={})
       end
     end
 
+    # Adds a listener to the array of listeners
+    #
+    # @param event [String] name of the event tha will be fired
+    # @param callback [String] callback function name for the event
+    # @return [Array] array of listeners
+    def add_listener(event, callback)
```

Comment:
Listener in this PR?

Reply:
The code to export the chart in PNG format must be inside the `ready` listener.